lib/ruby_lsp/listeners/document_link.rb

```diff
@@ -124,7 +124,7 @@ def extract_document_link(node)
         match = comment.location.slice.match(%r{source://.*#\d+$})
         return unless match
 
-        uri = T.cast(URI(match[0]), URI::Source)
+        uri = T.cast(URI(T.must(match[0])), URI::Source)
```

Comment:
Could this `T.must` cause problem when used with Prism `0.26`?

Reply:
I don't believe there are runtime errors for calling `T.must` on an untyped value. It's just a static check, which shouldn't impact users of the LSP.